NinchatSDKSwift/Implementations/Network/Requests/CloseSession.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 14.4.2020 Somia Reality Oy. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+
+import Foundation
+
+struct CloseSession: ServiceRequest {
+    typealias ReturnType = Empty
+    typealias BodyType = CloseSessionBody
+
+    private(set) var url: String = "https://api.luupi.net/v2/poll"
```

Comment:
What is this doing here? The close_session should be made via the *call* API, not the long-poll API - and the configured API server name should be used.